src/tests/ui/planner/sidepanel.ts

```diff
@@ -1,6 +1,7 @@
 import { $, ElementFinder, by} from 'protractor';
 import  *  as ui from './../../ui';
 import * as support from '../../support';
+import { BaseElement } from './../../ui';
```

Comment:
you can remove this import and use ui.BaseElement here https://github.com/fabric8-ui/fabric8-planner/pull/2698/files#diff-7a240b018c48e08143bb41f696f3ec9bR92